python/example/single_cell_recipe.py

```diff
@@ -59,15 +59,10 @@ def global_properties(self, kind):
 
 recipe = single_recipe(cell, [arbor.cable_probe_membrane_voltage('"midpoint"')])
 
-# (6) Create a default execution context and a default domain decomposition.
-
-context = arbor.context()
-domains = arbor.partition_load_balance(recipe, context)
-
 # (7) Create and run simulation and set up 10 kHz (every 0.1 ms) sampling on the probe.
 # The probe is located on cell 0, and is the 0th probe on that cell, thus has probe_id (0, 0).
 
-sim = arbor.simulation(recipe, domains, context)
+sim = arbor.simulation(recipe)
```

Comment:
These are examples. I'd like to show the verbose way at least once, and in this example in particular (it's where the `recipe` and `context` (and load balancers) are introduced.

To the accompanying tutorial page we can add how it could be shortened.

Reply:
Yes, introducing the long form is important!